### DISCUSSION TITLE
Add the plugin on the webpack.js.org

### DISCUSSION THREAD

**Discussion by yoriiis** [yoriiis/svg-chunk-webpack-plugin#10] [General]
@alexander-akait Do you think the plugin could be added on the [webpack documentation](https://webpack.js.org/plugins/) website?

I see that there is a `Community` section differentiates the plugins present in the `webpack-contrib` repository from others.
In the past, the section did not exist it seems to me when I submitted an other plugin (webpack/webpack.js.org#3771).

I am of course available if you think of changes that should be made to the project.

Thanks

**Comment by alexander-akait** [comment]
Yeah, of course, PR welcome :+1:

**Comment by yoriiis** [comment]
The plugin is available on webpack documentation https://webpack.js.org/plugins/svg-chunk-webpack-plugin
